observability/monitoring/managing-alerts.adoc

```diff
@@ -101,20 +101,35 @@ include::modules/monitoring-configuring-different-alert-receivers-for-default-pl
 // Creating alert routing for user-defined projects
 include::modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc[leveloffset=+2]
 
-// Applying a custom Alertmanager configuration
+[id="configuring-alertmanager-to-send-notifications"]
+== Configuring Alertmanager to send notifications
+
+You can configure Alertmanager to send notifications by editing the 
 ifndef::openshift-dedicated,openshift-rosa[]
-include::modules/monitoring-applying-custom-alertmanager-configuration.adoc[leveloffset=+1]
+`alertmanager-main` secret for default platform alerts or
 endif::openshift-dedicated,openshift-rosa[]
+`alertmanager-user-workload` secret for user-defined alerts.
 
-// Applying a custom configuration to Alertmanager for user-defined alert routing
-include::modules/monitoring-applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing.adoc[leveloffset=+1]
+[NOTE]
+====
+All features of a supported version of upstream Alertmanager are also supported in an OpenShift Alertmanager configuration. To check all the configuration options of a supported version of upstream Alertmanager, see: link:https://prometheus.io/docs/alerting/0.26/configuration/[Alertmanager configuration].
+====
+
+// Configuring notifications for default platform alerts
+ifndef::openshift-dedicated,openshift-rosa[]
+include::modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc[leveloffset=+2]
+endif::openshift-dedicated,openshift-rosa[]
+
+// Configuring notifications for user-defined alerts
+include::modules/monitoring-configuring-notifications-for-user-defined-alerts.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-.Additional resources
+[id="additional-resources_configuring-alertmanager-to-send-notifications"]
+== Additional resources
 
-* See link:https://www.pagerduty.com/[the PagerDuty official site] for more information on PagerDuty.
-* See link:https://www.pagerduty.com/docs/guides/prometheus-integration-guide/[the PagerDuty Prometheus Integration Guide] to learn how to retrieve the `service_key`.
-* See link:https://prometheus.io/docs/alerting/configuration/[Alertmanager configuration] for configuring alerting through different alert receivers.
+* link:https://www.pagerduty.com/[the PagerDuty official site] for more information on PagerDuty.
```

Comment:
```suggestion
* link:https://www.pagerduty.com/[PagerDuty official site]
```

Reply:
There's a move away from additional wording in AR links, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections